### ISSUE TITLE
non-zero code: 254 / Oracle home has not enough space when installing EE database

### ISSUE THREAD

**Issue by tkleiber** [oracle/docker-images#128]
Hi!

In which oracle home directory tries docker to install the ee database? The vagrant oel 7.2 box has 200 GB file system, and docker "Base Device Size: 53.69 GB". Where I have to look?

This is the output of running
./buildDockerImage.sh  -v 12.1.0.2 -e -i 

....
Checking Temp space: must be greater than 500 MB.   Actual 6780 MB    Passed
Checking swap space: 0 MB available, 150 MB required.    Failed <<<<

>>> Ignoring required pre-requisite failures. Continuing...

Preparing to launch Oracle Universal Installer from /tmp/OraInstall2016-07-26_06-48-19PM. Please wait ...[FATAL] [INS-32021] Insufficient disk space on this volume for the selected Oracle home.
   CAUSE: The selected Oracle home was on a volume without enough disk space.
   ACTION: Choose a location for Oracle home that has enough space (minimum of 6,553MB) or free up space on the existing volume.
A log of this session is currently saved as: /tmp/OraInstall2016-07-26_06-48-19PM/installActions2016-07-26_06-48-19PM.log. Oracle recommends that if you want to keep this log, you should move it from the temporary location.
Removing intermediate container a25bff2e6f4b
The command '/bin/sh -c unzip $INSTALL_FILE_1 &&     rm $INSTALL_FILE_1 &&        unzip $INSTALL_FILE_2 &&     rm $INSTALL_FILE_2 &&        $INSTALL_DIR/database/runInstaller -silent -force -waitforcompletion -responsefile $INSTALL_DIR/$INSTALL_RSP -ignoresysprereqs -ignoreprereq &&     rm -rf $INSTALL_DIR/database' returned a non-zero code: 254
There was an error building the image.

Kind regards
Torsten

**Comment by gvenzl** [comment]
The ORACLE_BASE and ORACLE_HOME are specified in the Dockerfile itself:

`ENV ORACLE_BASE=/opt/oracle \
    ORACLE_HOME=/opt/oracle/product/12.1.0.2/dbhome_1`

**Comment by tkleiber** [comment]
Thanks. I'm sure that this file system has round about 200 GB free, as I have added a disk for this to the vagrant virtualbox. Could this be related to the failing swap space check? Normally I add swapfile to the vagrant virtualbox, but have seen yesterday that this does not work when the file system is xfs.

**Comment by brunoborges** [comment]
The vbox may have space, but have you configured Docker to use that space? See issue #124.

Regardless, this is a problem we cannot solve at Dockerfile level.

**Comment by gvenzl** [comment]
It is not related to the swap space. The log snippet clearly indicates lack of space for the ORACLE_HOME itself:

    [FATAL] [INS-32021] Insufficient disk space on this volume for the selected Oracle home.
    CAUSE: The selected Oracle home was on a volume without enough disk space.
    ACTION: Choose a location for Oracle home that has enough space (minimum of 6,553MB) or free up space on the existing volume.

Please make sure that there is enough space available for the docker image and under /opt inside the docker image.

**Comment by tkleiber** [comment]
The disk size was big enough, as it was added with vagrant and formatted and added to the volume group via script before.
This I could verify wiht df -h too.
The changed volume group, which provides the /opt directory, had over 200 GB available.
As stated before I had changed docker base device size too before.

I have now corrected my script for providing swap space to my oel vagrant box.
The error was, that creating swapfile is not possible for all file systems.
On the mentionend volume group it was xfs, which is not usable for swapfile.
Have created it now on another volume group wiht ext4 file sytem succesfully.

After that the installation was successfully for 12c ee and 11g xe.

Thank you for your support.
